ui-admin/src/study/participants/KitRequests.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useMemo, useState } from 'react'
+import { Enrollee, KitRequest } from 'api/api'
+import { StudyEnvContextT } from '../StudyEnvironmentRouter'
+import { ColumnDef, getCoreRowModel, useReactTable } from '@tanstack/react-table'
+import { basicTableLayout } from 'util/tableUtils'
+import RequestKitModal from './RequestKitModal'
+import { faPlus } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import {instantToDefaultString} from "../../util/timeUtils";
+
+function KitRequestAddress({ sentToAddressJson }: { sentToAddressJson: string }) {
+  const address = JSON.parse(sentToAddressJson)
+  return <div>
+    <div>{address.firstName} {address.lastName}</div>
+    <div>{address.street1}</div>
+    {address.street2 && <div>{address.street2}</div>}
+    <div>{address.city}, {address.state} {address.postalCode} {address.country}</div>
+  </div>
+}
+
+export default function KitRequests({ enrollee, studyEnvContext, onUpdate }:
+                                      {
+                                        enrollee: Enrollee,
+                                        studyEnvContext: StudyEnvContextT,
+                                        onUpdate: () => void
+                                      }) {
+  const [showRequestKitModal, setShowRequestKitModal] = useState(false)
+
+  const onSubmit = () => {
+    setShowRequestKitModal(false)
+    onUpdate()
+  }
+
+  const columns = useMemo<ColumnDef<KitRequest, string>[]>(() => [{
+    header: 'Kit type',
+    accessorKey: 'kitType.displayName'
+  }, {
+    header: 'Status',
+    accessorKey: 'status'
+  }, {
+    header: 'Created',
+    accessorKey: 'createdAt',
+    accessorFn: data => instantToDefaultString(data.createdAt)
+  }, {
+    header: 'Address',
+    cell: ({ row }) => <KitRequestAddress sentToAddressJson={row.original.sentToAddress}/>
+  }, {
+    header: 'DSM Status',
+    accessorKey: 'dsmStatus'
+  }], [enrollee.kitRequests])
```

Comment:
yeah, this isn't the right dependency here -- nothing about the column definitions change with repsect to new data.  So this should be `[]` or, better yet, the declaration pulled outside the component